runtimes/rust/src/lib.rs

```diff
@@ -590,6 +590,19 @@ impl Runtime for RustRuntime {
         catch_panic(|| service.initialize(context, parameters))
     }
 
+    fn initiate_resuming_service(
```

Comment:
Should we document that core guarantees that this method always will be called with compatible and known artifact in `InstanceSpec`?

Reply:
@popzxc Yeah, documenting new functionality always makes sense. @alekseysidorov Do you plan to handle docs separately?

Reply:
I plan to write doc in this PR as soon as possible.